examples/example_detector_config_files/minimum_hexagon_config.json

```diff
@@ -0,0 +1,109 @@
+{
+	"name":"Simple Si Hexagon",
+	"units": {
+		"length": "mm",
+		"angle": "deg",
+		"potential": "V",
+		"temperature": "K"
+	},
+	"grid": {
+		"coordinates": "cartesian",
+		"axes": {
+            "x": {
+								"from": -2,
+                "to": 2,
+                "boundaries": {
+                    "left": "inf",
+                    "right": "inf"
+                }
+            },
+            "y": {
+                "from": -2,
+                "to": 2,
+								"boundaries": {
+                    "left": "inf",
+                    "right": "inf"
+                }
+            },
+            "z": {
+                "from": -1,
+                "to": 3,
+                "boundaries": {
+                    "left": "inf",
+                    "right": "inf"
+                }
+            }
+        }
+	},
+	"medium": "vacuum",
+	"objects": [
+		{
+			"type": "semiconductor",
+			"material": "Si",
+			"bulk_type": "n",
+			"temperature": 140,
+			"charge_density_model": {
+				"name": "linear",
+				"r": {
+					"init": 0.0,
+					"gradient": 1e-12
+				},
+				"phi": {
+					"init": 0.0,
+					"gradient": 0.0
+				},
+				"z": {
+					"init": 1e-12,
+					"gradient": 1e-13
+				}
+			},
+			"geometry": {
+				"type": "HexagonalPrism",
+				"r": {
+					"from": 0.0,
+					"to": 1.0
+				},
+				"h": 2.0,
+				"translate": {
+					"z": 1.0
+				}
+			}
+		},
+		{
+			"type": "contact",
+			"material": "Si",
+			"name": "n+ contact",
+			"potential": 0.0,
+			"channel": 1,
+			"geometry": {
+				"type": "HexagonalPrism",
+				"r": {
+					"from": 0.,
```

Comment:
Actually, one of the two contacts should have some inner radius > 0, e.g. 0.4. 
For reasons of testing.